DataStructures/LinkedList/SinglyLinkedList/SinglyLinkedList.cs

```diff
@@ -153,4 +153,56 @@ public bool DeleteElement(T element)
 
         return false;
     }
+
+    /// <summary>
+    /// Deletes the first element of the list.
+    /// </summary>
+    /// <returns> true if the operation is successul.</returns>
+    public bool DeleteFirst()
+    {
+        // checks if the List is empty
+        if(Head is null)
+        {
+            return false;
+        }
+
+        // if not, the head is overwritten with the next element and the old head is deleted
+        SinglyLinkedListNode<T>? temp = Head;
```

Comment:
I don't know if assigning the deleted node address to null helps with the garbage collection or not
please review it

Reply:
Assigning null to a variable is redundant. The reference will be lost as soon as it's no longer needed, so you don't need to help garbage collector.